src/module.d.ts

```diff
@@ -419,15 +420,7 @@ export namespace utils {
     const dataUtils: PropertyBag<Function>;
     const rowUtils: PropertyBag<Function>;
 
-    // TODO: Not a fan of this but was trying to avoid 
-    // wholesale copying the 11 or so types out of definitely-typed's connect
-    // not immediately seeing how to say for this thing, just reference their types
-    function connect(
-        mapStateToProps?: ReactRedux.MapStateToPropsParam<any, any>,
-        mapDispatchToProps?: ReactRedux.MapDispatchToPropsParam<any, any>,
-        mergeProps?: ReactRedux.MergeProps<any, any, any, any>,
-        options?: ReactRedux.Options
-    ) : ReactRedux.InferableComponentEnhancerWithProps<any, any>;
+    const connect : typeof originalConnect;
```

Comment:
This adjustment seems to have fixed the project I'm upgrading, but I'm unclear why this isn't equivalent to `typeof ReactRedux.connect`.